src/assets/css/index.css

```diff
@@ -1,6 +1,18 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,300;0,500;1,700;1,900&display=swap');
 
 * {
-  font-family: 'Poppins';
-  font-size: 100;
-}
\ No newline at end of file
+  margin: 0;
+  padding: 0;
```

Comment:
- [ ] You may consider moving the `margin` and `padding property inside the `body`  so that it won't modify the behavior of all elements
- [ ] I would suggest putting the `font-family` property under the `*` element  (OPTIONAL)